app/helpers/referrari/referrals_helper.rb

```diff
@@ -1,4 +1,9 @@
 module Referrari
   module ReferralsHelper
+
+    def testing
```

Comment:
Nope.  Maybe the file should just be deleted, unless there's a specific need for it.